index.bs

```diff
@@ -2581,6 +2581,37 @@ value explicitly. <em>block-size</em> is defined to be 128
 sample-frames which corresponds to roughly 3ms at a sample-rate of
 44.1 kHz.
 
+<h4 id="AudioNode-tail">AudioNode tail-time</h4>
+
+An {{AudioNode}} can have a <dfn>tail-time</dfn>. This means that even when the
+{{AudioNode}} is fed silence, non-silent output can be output.
+
+<h4 id="AudioNode-actively-processing">AudioNode lifetime</h4>
+
+{{AudioNode}} can be <dfn id="actively-processing">actively processing</dfn>
```

Comment:
Perhaps this can be:

An AudioNode accessed during a render quantum is referred to as actively processing. When an AudioNode is not actively processing, it MUST generate a single channel of silence as an output. This state of "actively processing" varies on the type of AudioNodes:

Reply:
I find this to be too vague. When we're checking whether an `AudioScheduledSource` is supposed to write something in its output, we're somehow accessing it, aren't we?

Reply:
Then how about "being rendered" instead of "accessed"?

Reply:
As a definition, this sentence basically ends without defining what actively processing.  Maybe say that it's actively processing if the any of the following conditions hold?  This probably also means that the links to "actively processing" shouldn't be links.